cloud/deploys/bin/azure/key-vault-setup

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env bash
+source lib/out.sh
+source azure/key-vault.sh
```

Comment:
suggestion (blocking): not everything in `azure/key-vault.sh` is specific to working with key vault, recommend renaming to `lib.sh` for now. Later we can refactor it into a directory with multiple files if need be.

Reply:
renamed